Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -761,10 +764,60 @@ public function testDbalOracleInstancename()
 
     public function testDbalSchemaFilter()
     {
+        if (method_exists(\Doctrine\DBAL\Configuration::class, 'setSchemaAssetsFilter')) {
+            $this->markTestSkipped('Test only applies to doctrine/dbal 2.9 or lower');
```

Comment:
Shouldn't this be 2.8 or lower since 2.9 contains the method?

```suggestion
            $this->markTestSkipped('Test only applies to doctrine/dbal 2.8 or lower');
```